### PR TITLE
Replayers > Relayers

### PR DESCRIPTION
**Pull request by chancefcc**
Oracle Replayers > Oracle Relayers

### CODE REVIEW

**Comment by Tweakwild113** [comment]
> Oracle Replayers > Oracle Relayers

So input this

**Comment by Tweakwild113** [comment]
Hello

**Review by Tweakwild113 — APPROVED** [review]

**Review by Mongkol67 — APPROVED** [review]

File: WHITEPAPER.md [hunk 1]
```diff
@@ -333,7 +333,7 @@ If some Relayers have faster networks and better hardware, they can monopolize a
 
 Relayers for BSC to BC communication are using the “Oracle” model, and so-called “**Oracle Relayers**”. Each of the validators must, and only the ones of the validator set, run Oracle Relayers. Each Oracle Relayer watches the blockchain state change. Once it catches Cross-Chain Communication Packages, it will submit to vote for the requests. After Oracle Relayers from ⅔ of the voting power of BC validators vote for the changes, the cross-chain actions will be performed.
 
-Oracle Replayers should wait for enough blocks to confirm the finality on BSC before submitting and voting for the cross-chain communication packages onto BC.
+Oracle Relayers should wait for enough blocks to confirm the finality on BSC before submitting and voting for the cross-chain communication packages onto BC.
 
 The cross-chain fees will be distributed to BC validators together with the normal BC blocking rewards.
```
Comment (Mongkol67) [inline WHITEPAPER.md:338]: #

File: WHITEPAPER.md [hunk 2]
```diff
@@ -333,7 +333,7 @@ If some Relayers have faster networks and better hardware, they can monopolize a
 
 Relayers for BSC to BC communication are using the “Oracle” model, and so-called “**Oracle Relayers**”. Each of the validators must, and only the ones of the validator set, run Oracle Relayers. Each Oracle Relayer watches the blockchain state change. Once it catches Cross-Chain Communication Packages, it will submit to vote for the requests. After Oracle Relayers from ⅔ of the voting power of BC validators vote for the changes, the cross-chain actions will be performed.
 
-Oracle Replayers should wait for enough blocks to confirm the finality on BSC before submitting and voting for the cross-chain communication packages onto BC.
```
Comment (Mongkol67) [inline WHITEPAPER.md:336]: -

**Review by Mongkol67 — CHANGES_REQUESTED** [review]

**Review by Mongkol67 — APPROVED** [review]

**Review by Mongkol67 — APPROVED** [review]

**Review by Mongkol67 — APPROVED** [review]

**Review by doctorhamed — APPROVED** [review]

File: WHITEPAPER.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Floyd6531) [reply WHITEPAPER.md:336]: Facts

[the remark this answers is not in the document]
Comment (dammywise12) [reply WHITEPAPER.md:336]: Ok

[the remark this answers is not in the document]
Comment (dammywise12) [reply WHITEPAPER.md:336]: Ok

**Review by dammywise12 — APPROVED** [review]

File: WHITEPAPER.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alice22foster) [reply WHITEPAPER.md:336]: 0xd700a72b077d1bc1d242f3b354dc6d0d9459af7e7e2553d3d8c9d4c0a829509a

**Review by alice22foster — APPROVED** [review]

**Pull request closed without merging**